README.md

```diff
@@ -10,8 +10,9 @@
 Pycytominer is a suite of common functions used to process high dimensional readouts from high-throughput cell experiments.
 The tool is most often used for processing data through the following pipeline:
 
-<img height="500" src="https://github.com/cytomining/pycytominer/blob/master/media/pipeline.png?raw=true">
-[Click here for high resolution image](https://github.com/cytomining/pycytominer/blob/master/media/pipeline.png)
+<img height="325" src="https://github.com/cytomining/pycytominer/blob/master/media/pipeline.png?raw=true">
```

Comment:
Consider adding an `alt` attribute to this `img` which helps describe the image to increase web accessibility.

Reply:
good point, will add